aten/src/ATen/cuda/tunable/GemmCommon.h

```diff
@@ -85,15 +85,15 @@ struct GemmParams : OpParams {
   }
 
   size_t GetSizeA() const {
-    return sizeof(T) * lda * ((transa == 'n' || transa == 'N') ? k : m);
+    return sizeof(T) * m * k;
```

Comment:
noob q, we don't need to worry about transpose any more?

Reply:
This fix was too brash.  Expect something better.  We're writing more unit tests with this PR to capture all cases and this change here breaks when you have a 4D tensor sliced down to 2D.

Reply:
Resolved.  The original size calcs were correct for standard gemm.  Only the strided batched was wrong.  Updated, with a new unit test to cover.  PR is minimal diff now.